.github/workflows/main.yml

```
@@ -21,9 +26,22 @@ jobs:
         DESTDIR: out
       run: |
         meson build
-        ninja -C build
         ninja -C build install
-              
+
+    - name: Build (Fedora)
+      env:
+        DESTDIR: out
+      run: |
+        meson configure -Dcurated=false -Dhomepage=false -Dpayments=false  build
+        ninja -C build install
+
+    - name: Build (Pop!_Shop)
+      env:
+        DESTDIR: out
+      run: |
+        meson configure -Dcurated=false -Dpayments=false -Dsharing=false build
```

Should also test with `name` build flag, even though it _shouldn't_ affect building.

```suggestion
        meson configure -Dcurated=false -Dpayments=false -Dsharing=false -Dname=Pop\!_OS build
```